### PR TITLE
fix analysis issues

### PR DESCRIPTION
**Pull request by grouma**

### CODE REVIEW

**Comment by natebosch** [comment]
@kevmoo  - maybe we shouldn't be changing for bleeding edge SDK before it's imported to google?

**Comment by grouma** [comment]
Current SDK inside Google3 is "v2_0_0_dev_8_0". What version of the SDK did @kevmoo's change require?

File: lib/src/frontend/expect.dart [hunk 1]
```diff
@@ -157,6 +157,9 @@ Future _expect(actual, matcher,
     reason ??= '$e at $trace';
   }
   fail(formatter(actual, matcher, reason, matchState, verbose));
+  // Fix missing return. The above fail() will always throw, but static
+  // analysis doesn't know this.
+  throw "Unreachable";
```
Comment (kevmoo) [inline lib/src/frontend/expect.dart]: Add an `ignore: ...xyz` here for the dev SDK where this is reported...maybe w/ a todo?

**Comment by kevmoo** [comment]
We figured it out -dev.12 I believe

**Comment by nex3** [comment]
I don't want to regress analysis on bleeding edge. I'd rather temporarily have changes in Google3 only while we're waiting for the SDK to be rolled in than temporarily have changes externally as well.

**Comment by kevmoo** [comment]
Google3 is now at -dev.18 – should be good, right?

**Comment by kevmoo** [comment]
...or just delete the branch entirely, @grouma ?

**Pull request closed without merging**